pkg/deviceplugin/implementation.go

```diff
@@ -28,28 +29,69 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	initialDevicesQuantity = 15
+	// the maximum pods per node are 256,
+	// so this number should be more than enough
+	devicesLimit = 1024
+)
+
+type message struct {
+	requestedDevices int
+}
+
 type pluginImp struct {
-	mutualCpus *cpuset.CPUSet
+	mutualCpus       *cpuset.CPUSet
+	update           chan message
+	allocatedDevices int
 }
 
 func (p pluginImp) ListAndWatch(empty *pluginapi.Empty, server pluginapi.DevicePlugin_ListAndWatchServer) error {
-	devs := MakeMutualCpusDevices()
-	glog.V(4).Infof("ListAndWatch respond with: %+v", devs)
-	if err := server.Send(&pluginapi.ListAndWatchResponse{Devices: devs}); err != nil {
+	var devID int
+	devs := makeDevices(initialDevicesQuantity, devID)
+	devID += initialDevicesQuantity
```

Comment:
not-so-minor: it seems easier and less error prone to just do
```
devs := makeDevices(initialDevicesQuantity, devID)
devID += len(devs)
```
any reason not to do this?

Reply:
Good idea, i'll fix that